Update kr.m3u

Alstruit opened the pull request:
Original Reference #9033:
Closed for a more clean history.

To implement:
Convert seeZN links to public links.
Append seeZN links 1080p by @LaneSh4d0w  scrapper tool.

Alstruit committed:
Remove channels

Contains self hosted sites (Tvheadend).

Alstruit committed:
Reword Title

Alstruit committed:
Add channel

LaneSh4d0w commented:
you'll need the channel number over Seezn's data.
285 is Mnet
273 is JTBC
213 is AsiaN

Alstruit committed:
Remove broken links

Alstruit committed:
Fix channels

Remove non working links.
Replaced broken unique links with @LaneSh4dow scrapper.
Reasons for removal:
Restricted content
Not 24/7 for an indefinite time. (Events).
Alternative links that will be or have been replaced.

Alstruit committed:
Add more links

A few Ids need to be placed.

Alstruit commented:
Some channels from Seezn can be temporarly blocked at any time. Not sure if the 5 minute check is consistent.

LaneSh4d0w commented:
> Some channels from Seezn can be temporarly blocked at any time. Not sure if the 5 minute check is consistent.

I think i'll need to put my link inside the playlist to recreate a m3u file for each session. This would prevent the program cutting off unexpectedly.

Alstruit committed:
Replace link with alternative

freearhey commented on streams/kr.m3u:
```diff
@@ -325,3 +279,67 @@ http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index.m3u8
+#EXTINF:-1 tvg-id="ArtTVArte.kr" status="",Arte TV (1080p)
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCPglj7zGwmOlH8_31Tpgh8Q/live
+#EXTINF:-1 tvg-id="BravoKids.kr" status="",Bravo Kids (1080p)
+https://www.youtube.com/channel/UCu6mog947IWOXL23AzKlImw/live
```
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCu6mog947IWOXL23AzKlImw/live
```

freearhey commented on streams/kr.m3u:
```diff
@@ -325,3 +279,67 @@ http://necgokr2-724.acs.wecandeo.com/ms/2528/724/index.m3u8
+#EXTINF:-1 tvg-id="LotteHomeShopping.kr" status="",Lotte Home Shopping (720p)
+https://pchlslivesw.lotteimall.com/live/livestream/lotteimalllive_mp4.m3u8
+#EXTINF:-1 tvg-id="" status="",Maeil Business Newspaper TV (매일경제 TV) (1080p)
+https://www.youtube.com/channel/UCnfwIKyFYRuqZzzKBDt6JOA/live
```
```suggestion
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCnfwIKyFYRuqZzzKBDt6JOA/live
```

Alstruit committed:
Apply suggestions from code review

Co-authored-by: Aleksandr Statciuk <free.arhey@gmail.com>

LaneSh4d0w commented on streams/kr.m3u:
```diff
@@ -113,30 +109,26 @@ http://mgugaklive.nowcdn.co.kr/gugakvideo/gugakvideo.stream/playlist.m3u8
+https://kr-streams.onrender.com/scraper?channel=523
+#EXTINF:-1 tvg-id="JTBCGolf.kr" status="",JTBC Golf (1080p)
+https://kr-streams.onrender.com/scraper?channel=299
+#EXTINF:-1 tvg-id="JTBCGolfSports.kr" status="",JTBC Golf & Sports (480p) [Not 24/7] [Geo-locked]
```
```suggestion
#EXTINF:-1 tvg-id="JTBCGolfSports.kr" status="",JTBC Golf & Sports (480p) [Not 24/7] [Geo-blocked]
```

BellezaEmporium committed:
Update streams/kr.m3u

Alstruit committed:
Merge branch 'master' into alstruit-9.26_kr

Alstruit committed:
Remove Duplicates

Alstruit committed:
Merge branch 'master' into alstruit-9.26_kr

MapGuy11 committed:
Merge branch 'master' into alstruit-9.26_kr

Alstruit committed:
Merge branch 'master' into alstruit-9.26_kr

Alstruit committed:
Merge branch 'master' into alstruit-9.26_kr

Red3amer reviewed (commented):
``

Alstruit committed:
Merge branch 'master' into alstruit-9.26_kr
